lib/ts/recipe/session/sessionClass.ts

```diff
@@ -71,11 +76,18 @@ export default class Session implements SessionContainerInterface {
 
     updateSessionData = async (newSessionData: any, userContext?: any) => {
         try {
-            await this.helpers.sessionRecipeImpl.updateSessionData({
-                sessionHandle: this.sessionHandle,
-                newSessionData,
-                userContext: userContext === undefined ? {} : userContext,
-            });
+            if (
+                !(await this.helpers.sessionRecipeImpl.updateSessionData({
+                    sessionHandle: this.sessionHandle,
+                    newSessionData,
+                    userContext: userContext === undefined ? {} : userContext,
+                }))
+            ) {
+                throw new STError({
+                    message: "Session does not exist anymore.",
```

Comment:
This error message is different from the rest. Also, I think this is a better error message, since the lifetime of the "session information" is the same as the lifetime of the entire session, right?